rwgame/WindowIcon.h

```diff
@@ -0,0 +1,4103 @@
+0xb9,0x48,0x70,0xff,0xb8,0x48,0x70,0xff,0xb8,0x48,0x70,0xff,0xb8,0x48,0x70,0xff,
+0xb8,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,
+0xb7,0x47,0x70,0xff,0xb6,0x47,0x70,0xff,0xb6,0x47,0x70,0xff,0xb6,0x46,0x70,0xff,
+0xb6,0x46,0x70,0xff,0xb5,0x46,0x70,0xff,0xb4,0x46,0x70,0xff,0xb6,0x46,0x72,0xff,
+0xbe,0x49,0x76,0xff,0x5c,0x23,0x38,0xff,0x02,0x00,0x02,0xff,0x00,0x00,0x00,0xff,
+0x00,0x00,0x00,0xff,0x00,0x00,0x00,0xff,0x00,0x00,0x00,0xd8,0x00,0x00,0x00,0x4c,
+0x00,0x00,0x00,0x03,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
+0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
+0x00,0x00,0x00,0x13,0x00,0x00,0x00,0x91,0x00,0x00,0x00,0xf9,0x00,0x00,0x00,0xff,
+0x00,0x00,0x00,0xff,0x14,0x06,0x05,0xff,0x65,0x1d,0x16,0xff,0x77,0x23,0x1a,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,
+0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x72,0x22,0x19,0xff,0x7a,0x24,0x1a,0xff,
+0x57,0x19,0x13,0xff,0x0d,0x03,0x03,0xff,0x00,0x00,0x00,0xff,0x00,0x00,0x00,0xff,
+0x02,0x01,0x01,0xff,0x3a,0x25,0x15,0xff,0xd2,0x84,0x4c,0xff,0xf7,0x9b,0x5a,0xff,
+0xe8,0x92,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,
+0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xea,0x93,0x55,0xff,0xe9,0x92,0x55,0xff,
+0xf4,0x98,0x58,0xff,0xd0,0x82,0x4c,0xff,0x2d,0x1c,0x10,0xff,0x00,0x00,0x00,0xff,
+0x00,0x00,0x00,0xff,0x00,0x00,0x00,0xff,0x04,0x02,0x03,0xff,0x57,0x21,0x33,0xff,
+0xc2,0x4b,0x73,0xff,0xc1,0x4b,0x72,0xff,0xbb,0x49,0x6f,0xff,0xbb,0x49,0x6f,0xff,
+0xbb,0x48,0x6f,0xff,0xba,0x48,0x6f,0xff,0xba,0x48,0x6f,0xff,0xba,0x48,0x6f,0xff,
+0xba,0x48,0x6f,0xff,0xb9,0x48,0x70,0xff,0xb8,0x48,0x70,0xff,0xb9,0x48,0x70,0xff,
+0xb9,0x47,0x70,0xff,0xb8,0x47,0x70,0xff,0xb8,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,
+0xb7,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,0xb7,0x47,0x70,0xff,0xb8,0x48,0x71,0xff,
+0xc0,0x4a,0x76,0xff,0x5e,0x24,0x39,0xff,0x02,0x
```

Comment:
It's been generated from what GIMP gave me, "GIMP RGBA C-Source image dump" is the format.
I assume it has that last byte in it to signal the end of the buffer.

```
#include <stdio.h>
#include <stdlib.h>

#define COL 16

#include "../openrw/rwgame/WindowIcon.h"

void main()
{
    int i = 0;
    int size = sizeof(windowIconData);

    while (i < size) {
        if ((i > 0) && (i % COL == 0)) {
            printf("\n");
        }

        printf("0x%02x", windowIconData[i]);
        printf(",");

        i++;
    }

    puts("");
}
```
is what I used to convert it to an array of bytes.